idd/Energy+.idd.in

```diff
@@ -16508,57 +16519,50 @@ SurfaceProperty:ExposedFoundationPerimeter,
        \minimum 0.0
        \maximum 1.0
        \default 1.0
```

Comment:
The limits and default here look more appropriate for Total Exposed Perimeter Fraction.  You really need to add another field here by that name.  
Total Exposed Perimeter Fraction (0:1) and Total Exposed Perimeter (0:nolimit) in {m}